index.html

```diff
@@ -4,6 +4,12 @@
   <title>HTML Syntax</title>
 </head>
 <body>
+<html>
```

Comment:
This `index.html` file is an extra, please delete it. You can keep the one in the `html-lists` folder